scripts/update_schemas.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
```

Comment:
What do you think of creating a TextCommand that could be triggered from the command palette. Example `LSP-json: Update schemas` and that command would run this code bellow?

This can be an improvement for later, if you want to. 
But am happy that you took the time to create a script :)

Reply:
I don't think they change often enough for it to be worth it. Also, since the package is compressed, we would have to save to cache folder. And also we would need to add dependency on requests package also. But my main argument is that I think it's not worth it.